ts/nni_manager/training_service/reusable/environment.ts

```diff
@@ -134,6 +133,12 @@ export abstract class EnvironmentService {
         return 0;
     }
 
+    public abstract get getPlatform(): string;
+
+    public get getCommandChanneName(): Channel {
```

Comment:
Typo:
getCommandChanneName -> getCommandChannelName

But Why get name is enough? It's better to rename `createCommandChannel` to `initializeCommandChannel`, and then keep the created channel object in the environment service.

Reply:
fixed typo. Use get name because trialDispatcher has a map to maintain the command channels. The original purpose to store command channel in Map not in environment service is to keep different type of command channel singleton instance when intialize them.

Reply:
For WebChannel, it can be a single instance, but maintained by WebChannel itself. For FileChannel, most likely it's different for each service. I'm not sure about AMLChannel, if there are two AML instances.

Reply:
updated, use singleton pattern for webchannel.